zeppelin-plugins/launcher/k8s-standard/pom.xml

```diff
@@ -37,21 +37,47 @@
 
     <properties>
         <plugin.name>Launcher/K8sStandardInterpreterLauncher</plugin.name>
+        <version.kubernetes.client>4.10.2</version.kubernetes.client>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-client</artifactId>
+            <version>${version.kubernetes.client}</version>
```

Comment:
how about `kubernetes.client.version` ? This is the convention we use for other libraries.

Reply:
accept the proposal